pkg/webui/components/form/section/index.js

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import classnames from 'classnames'
+import { defineMessages, useIntl } from 'react-intl'
+
+import Form, { useFormContext } from '@ttn-lw/components/form'
+import Icon from '@ttn-lw/components/icon'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import style from './section.styl'
+
+const m = defineMessages({
+  expand: 'Expand section',
+  collapse: 'Collapse section',
+})
+
+// `<FormCollapseSection />` aggregates a set of form fields under common title as well as adds
+// functionality to hide or show the fields.
+const FormCollapseSection = props => {
+  const { className, id, title, onCollapse, initiallyCollapsed, children } = props
```

Comment:
Just curious, why do you require an `id`?

Reply:
for the `aria-controls` attribute